src/databricks/labs/ucx/mounts/list_mounts.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from dataclasses import dataclass
+
+from databricks.sdk import WorkspaceClient
+
+# Added a * import otherwise it won't be possible to import spark directly
+from databricks.sdk.runtime import *  # noqa: F403
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class MountResult:
+    name: str
+    source: str
+    instance_profile: str | None = None
+
+
+class MountLister:
+    def __init__(self, ws: WorkspaceClient, inventory_database: str):
+        self._ws = ws
+        self._inventory_database = inventory_database
+
+    def inventorize_mounts(self):
+        mounts = dbutils.fs.mounts()  # noqa: F405
+        print(f"found {len(mounts)} mount points in this workspace")
+        df = spark.createDataFrame(dbutils.fs.mounts()).selectExpr("mountPoint as name", "source")  # noqa: F405
+
+        target_table = f"{self._inventory_database}.mounts"
+        df.writeTo(target_table).replace()
+        print(f"All workspace mounts points saved to {target_table}")
```

Comment:
```suggestion
class Mounts(CrawlerBase):
    def __init__(self, backend: SqlBackend, ws: WorkspaceClient, inventory_database: str):
        super().__init__(backend, "hive_metastore", inventory_database, "mounts")
        self._dbutils = ws.dbutils

    def inventorize_mounts(self):
        mounts = []
        for mount_point, source, _ in self._dbutils.fs.mounts():
            mounts.append(MountResult(mount_point, source))
        self._append_records(mounts)
```

and then call it from task as `Mounts(RuntimeBackend(), WorkspaceClient(config=cfg.as_workspace_config()), cfg.inventory_database).inventorize_mounts()` (or a bit more readable way =P)

then you can integration test it by

```
commands = CommandExecutor(ws)

    commands.install_notebook_library(f"/Workspace{wsfs_wheel}")
    commands.run(
        """
        from databricks.labs.ucx.hive_metastore.mounts import Mounts
        backend = RuntimeBackend()
        Mounts(...)
        ...
        """
    )
```

and you can mock it for unit testing.